notebooks/Revised_Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -0,0 +1,1528 @@
+   "source": [
+    "Here, the theta and phi values are set using **set_theta_phi()** using the theta and phi values of the GRB source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 88,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 48 µs, sys: 55 µs, total: 103 µs\n",
+      "Wall time: 107 µs\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "sig_mod.set_theta_phi(theta, phi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Diffuse and Point Source Models**\n",
+    "---\n",
+    "\n",
+    "The purpose of building a model is to eventually calculate the expected number of counts detected per detector and per energy bin as a function of our set of parameters. With these expected counts, they can be compared to the actual counts detected with the likelihood analysis. The models are split between the diffuse and point source models, which shall be described below.\n",
+    "\n",
+    "### **Diffuse Model**\n",
+    "The diffuse model describes the expected distribution of counts $\\lambda_{ij}$ for detector i and energy bin j:\n",
+    "\n",
+    "$$\\lambda_{ij}^{diff} = (\\Omega_i\\phi^{b}_{j} + r^{b}_{j})T$$\n",
+    "\n",
+    "where $\\Omega_i$ is the unblocked solid angle for detector i, $\\phi^{b}_{j}$ is the rate per detector solid angle, and $r^{b}_{j}$ is the rate per detector. \n",
+    "\n",
+    "### **Point Source Model**\n",
+    "The point source model is somewhat different in that it is dependent on the source's positions ($\\phi$, $\\theta$). It likewise describes the expected distribution of counts for the ith detector and jth energy bin as:\n",
+    "\n",
+    "$$\\lambda_{ij}^{PS} = T\\sum_{l}DRM_{ijl}(\\theta,\\phi)f(E_{\\gamma} = E_{\\gamma,l})\\Delta E_{\\gamma, l}$$\n",
+    "\n",
+    "where T is the exposure, $DRM_{ijl}(\\theta, \\phi)$ is the detector response matrix for the particular source position of $(\\theta, \\phi)$, $f(E_{\\gamma} = E_{\\gamma,l})$ is the photon spectrum in energy bin $E_{\\gamma, l}$, and (what is $\\Delta E_{\\gamma, l}$?)\n",
+    "\n",
+    "For more information, refer to sections 4.1.1 and 4.1.2."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Background Model**\n",
+    "---\n",
+    "\n",
+    "The background model encompasses any of the non-GRB sources that show up in the image. This can include contributions from the cosmic ray background or any number of sources that are known to be sources of cosmic rays. \n",
+    "\n",
+    "The background model consists mainly of a diffuse model from the contributions from the cosmic ray background combined with some point source models for each of the known sources. (sections 4.1.1 and 4.1.2) \n",
+    "\n",
+    "A collection of these sources is stored in the bkg_estimation.csv file. The path to this file is stored bkg_fname.\n",
```

Comment:
All the bright point sources that we consider for bkg modelling are stored here https://github.com/Swift-BAT/NITRATES/blob/main/bright_src_cat.fits, the estimation of bkg for the 4 sources relevant (in the field of view) for this particular burst are in bkg_estimation.csv